service/src/main/kotlin/fi/espoo/evaka/reservations/ReservationControllerCitizen.kt

```diff
@@ -285,6 +285,8 @@ private fun placementDay(
     val operationDays =
         backupPlacements?.find { it.range.includes(date) }?.operationDays ?: placement.operationDays
 
+    // Backup placements take precedence
+    val operationTime = (backupPlacements?.find { it.range.includes(date) }?.operationTimes ?: placement.operationTimes)[date.dayOfWeek.value - 1]
```

Comment:
`backupPlacements?.find { it.range.includes(date) }` is used twice and should be extracted as a variable

Reply:
Restructured accordingly.